src/cmds/scripts/pbs_dataservice

```diff
@@ -323,8 +332,23 @@ case "$1" in
 			fi
 			start_dataservice $1
 			ret=$?
-			if [ ${ret} -ne 0 -a "$2" != "PBS" ]; then
-				echo "Failed to start PBS Data Service"
+			if [ ${ret} -ne 0 ]; then
+				if  [ "$2" != "PBS" ]; then
+					echo "Failed to start PBS Data Service"
+				elif [ ${ret} -eq ${version_mismatch} ]; then
+					if [ -f "${log_file}" ]; then
+						cat $log_file
```

Comment:
Purpose of this log file is to just print the error message? If so why not store the message in a variable instead?

Reply:
this is the log file generated by pg_ctl. As PBS would need to display the logs when an error occurs, I am using cat so that it goes to the output of this script.